.github/workflows/pytest.yml

```
@@ -42,23 +46,21 @@ jobs:
           --health-retries 5
 
     steps:
-      - uses: actions/checkout@v3
-
-      - name: Install system dependencies
-        run: apt-get install -y libpq-dev
+       - name: Install and configure poetry
+        run: |
+          pipx install poetry==1.5.0
```

There is a syntax error, the indentation has one extra space compared to the next items in the `steps` array:
```suggestion
      - name: Install and configure poetry
        run: |
          pipx install poetry==1.5.0
```

---

Syntax error has been corrected as well